frameworks/g-cloud-12/questions/services/virtualisationTechnologiesUsedOther.yml

```diff
@@ -12,8 +12,8 @@ max_length_in_words: 100
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter details of the other virtualisation technology you use.
   - name: under_100_words
-    message: Your description must be no more than 100 words.
+    message: Description must be 100 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?